<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PuzzleClass</class>
 <widget class="QMainWindow" name="PuzzleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Puzzle</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>选择文件</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>重新开始(F5)</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>难度设置</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>查看原图</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="puzzle.qrc"/>
 </resources>
 <connections/>
</ui>
